.azure-pipelines/weekly-preview-generation.yml

```
@@ -13,7 +13,6 @@ parameters:
   - name: SkipForceRefresh
     displayName: Skip Force Refresh
     default: false
```

Type is needed for the parameter:
```suggestion
    default: false
    type: boolean
```